src/explorer.ts

```diff
@@ -333,14 +333,7 @@ class KubernetesSelectorResource extends KubernetesResource {
         if (!this.selector) {
             return [];
         }
-        let selectorString = "";
-        if (this.selector.matchLabels) {
-            const selectors = [];
-            for (const sel in this.selector.matchLabels) {
-                selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
-            }
-            selectorString = `-l ${selectors.join(",")}`;
-        }
+        const selectorString = kubectlUtils.getResourceSelectorString(this.selector);
```

Comment:
I'm getting lost in these changes I'm afraid.  It now looks like `getPods` isn't called from _anywhere_ so I'm not sure of the value of updating it.  And `KubernetesSelectorResource.getChildren` calling `kubectl.fromLines("get po...")` will return a _different schema_ from what `getPods` used to return.

It seems like we probably want to reset `KubernetesSelectorResource` and `getPods` to what they used to be except with an additional status field.  And perhaps we need to modify `KubernetesNodeResource.getChildren` to call `getPods` with the `all` namespace option.  That would keep us closest to how I think we were before we added the status GUI.